nixos/modules/services/misc/ollama.nix

```diff
@@ -3,12 +3,19 @@ let
   inherit (lib) types;
 
   cfg = config.services.ollama;
-  ollamaPackage = cfg.package.override {
+
+  stableOverrides = {
```

Comment:
I'd still have preferred `optionalAttrs` over independent branches, something like

```
let
  newArgs = prevArgs: {
    # ...
  } // lib.optionalArgs (prevArgs ? linuxPackages) {
    # ...
  }
in cfg.package.override newArgs
```

...but that's your call

Reply:
I'd definitely prefer that too.